README.md

```diff
@@ -2,7 +2,7 @@
 
 # Strawberry GraphQL
 
-> Python GraphQL library based on dataclasses
+> Python GraphQL library based on data classes
```

Comment:
```suggestion
> Python GraphQL library based on dataclasses
```

Here we are referring to the dataclasses module: https://docs.python.org/3/library/dataclasses.html